src/Provider/Node/SystemNodeProvider.php

```diff
@@ -88,17 +88,16 @@ protected function getSysfs()
     {
         $mac = false;
 
-        if (strtoupper(php_uname('s')) === "LINUX") {
+        if (strtoupper(php_uname('s')) === 'LINUX') {
             $addressPaths = glob('/sys/class/net/*/address', GLOB_NOSORT);
 
             if (empty($addressPaths)) {
                 return false;
             }
 
-            $macs = array_map(
-                'file_get_contents',
-                $addressPaths
-            );
+            array_walk($addressPaths, function ($addressPath) use (&$macs) {
```

Comment:
Does `array_walk()` perform better than `array_map()` or was this change made because it's easier to mock the functions and test them?

Reply:
It was done because of the mock. `array_map` performs slightly better (see [the related benchmark](https://gist.github.com/Potherca/51c7b6409f4d947c3bd7684c9cf0d96b)).